fdbclient/StorageServerInterface.h

```diff
@@ -423,9 +423,30 @@ struct SplitMetricsRequest {
 	}
 };
 
+// Should always be used inside a `Standalone`.
+struct ReadHotRangeWithMetrics {
```

Comment:
Maybe something like `ReadHotRangeRefWithMetrics`? But it does not actually matter that much since it's only usage is this one use case and I don't think it'd be reused somewhere else. And compare with adding `Ref` into its name, I'd still prefer this inline doc.